File: New_NativeUI/PauseMenu/Tabs/TabTextItem.cs
using CitizenFX.Core;
using NativeUI.PauseMenu.Items;
using System.Collections.Generic;
using System.Drawing;

namespace NativeUI.PauseMenu
{
    public class TabTextItem : BaseTab
    {
        public string TextTitle { get; set; }
        public string Text { get; set; }
        public int WordWrap { get; set; }
        public List<BasicTabItem> LabelsList = new List<BasicTabItem>();

        public TabTextItem(string name, string title) : base(name)
        {
            TextTitle = title;
        }

        public async void AddTitle(string title)
        {
            if(string.IsNullOrWhiteSpace(TextTitle))
                TextTitle = title;
        }

        public async void AddItem(BasicTabItem item)
        {
            LabelsList.Add(item);
        }
    }
}

File: ScaleformUI_Lua/src/PauseMenu/Tabs/TabTextItem.lua
TabTextItem = setmetatable({}, TabTextItem)
TabTextItem.__index = TabTextItem
TabTextItem.__call = function()
    return "BaseTab", "TabTextItem"
end

function TabTextItem.New(name, _title)
    data = {
        Base = BaseTab.New(name or ""),
        Label = name,
        TextTitle = _title or "",
        LabelsList = {},
        Index = 0,
        Focused = false,
        Parent = nil
    }
    return setmetatable(data, TabTextItem)
end

function TabTextItem:AddTitle(title)
    if not title:IsNullOrEmpty() then
        self.TextTitle = title
    end
end

function TabTextItem:AddItem(item)
    table.insert(self.LabelsList, item)
end
